applications/GeoMechanicsApplication/tests/test_compressibility.py

```diff
@@ -23,12 +23,12 @@ def test_compressibility_upw_small_strain_fic(self):
 
     def test_compressibility_upw_updated_lagrange_fic(self):
         test_name = 'compressibility_tests/upw_updated_lagrange_fic'
-        expected_water_pressure_at_bottom = -55.9999
+        expected_water_pressure_at_bottom = -55.6957
```

Comment:
Huh?
This is a surprise. removing duplication should normally not lead to changes in results.

Reply:
What I did for the test is actually set the "CONSIDER_GEOMETRIC_STIFFNESS" to true in the material parameters for the updated lagrange tests, since there were no tests which actually added the geometric stiffness. I ran these tests using the 'master' version of the code, revalidated the results for updated lagrangian, since I expected the results to be different, and used the test to make sure I didn't break anything for these tests.